v2/internal/controllers/crd_servicebus_namespace_standard_test.go

```diff
@@ -64,6 +69,48 @@ func NewServiceBusNamespace(tc *testcommon.KubePerTestContext, rg *resources.Res
 	return namespace
 }
 
+func ServiceBus_AuthorizationRule_CRUD(tc *testcommon.KubePerTestContext, sbNamespace client.Object) {
+	authSecret := "authsecret"
+
+	rule := &servicebus.NamespacesAuthorizationRule{
```

Comment:
AFAIK, there is an autogenerated one of these, called `RootManageSharedAccessKey`, which contains the root access keys.

How does the user get access to those?

Do they create a resource in ASO named `RootManageSharedAccessKey` and then export keys from it? Or do we need special handling for that named AccessPolicy because they forbid PUTs on it or something?

I think we should make sure to test access to that as well, as it will be a common ask

Reply:
Thanks for the heads-up and the offline discussion. I'll expose those details as secrets available on the Namespace itself.

Reply:
Done.